docs/source/contingency_tables.rst

```diff
@@ -0,0 +1,296 @@
+
+The underlying population for a contingency table is described by a
+**distribution table** :math:`P_{i, j}`.  The elements of :math:`P`
+are probabilities, and the sum of all elements in :math:`P` is 1.
+Methods for analyzing contingency tables use the data in :math:`T` to
+learn about properties of :math:`P`.
+
+The :class:`statsmodels.stats.Table` is the most basic class for
+working with contingency tables.  We can create a ``Table`` object
+directly from any rectangular array-like object containing the
+contingency table cell counts:
+
+.. ipython:: python
+
+    import numpy as np
+    import pandas as pd
+    import statsmodels.api as sm
+
+    df = sm.datasets.get_rdataset("Arthritis", "vcd").data
+
+    tab = pd.crosstab(df['Treatment'], df['Improved'])
+    table = sm.stats.Table(tab)
+
+Alternatively, we can pass the raw data and let the Table class
+construct the array of cell counts for us:
+
+.. ipython:: python
+
+    table = sm.stats.Table.from_data(df[["Treatment", "Improved"]])
+
+
+Independence
+------------
+
+**Independence** is the property that the row and column factors occur
+independently. **Association** is the lack of independence.  If the
+joint distribution is independent, it can be written as the outer
+product of the row and column marginal distributions:
+
+.. math::
+
+P_{ij} = \sum_k P_{ij} \cdot \sum_k P_{kj} \forall i, j
+
+We can obtain the best-fitting independent distribution for our
+observed data, and then view residuals which identify particular cells
+that most strongly violate independence:
+
+.. ipython:: python
+
+    print(table.table_orig)
+    print(table.fittedvalues)
+    print(table.pearson_resids)
+
+In this example, compared to a sample from a population in which the
+rows and columns are independent, we have too many observations in the
+placebo/no improvement and treatment/marked improvement cells, and too
+few observations in the placebo/marked improvement and treated/no
+improvement cells.  This reflects the apparent benefits of the
+treatment.
+
+If the rows and columns of a table are unordered (i.e. are nominal
+factors), then the most common approach for formally assessing
+independence is using Pearson's :math:`\chi^2` statistic.  It's often
+useful to look at the cell-wise contributions to the :math:`\chi^2`
+statistic to see where the evidence for dependence is coming from.
+
+.. ipython:: python
+
+    rslt = table.nominal_association
+    print(rslt.pvalue)
+    print(table.chi2_contribs)
+
+For tables with ordered row and column factors, we can us the **linear
+by linear** association test to obtain more power against alternative
+hypotheses that respect the ordering.  The test statistic for the
+linear by linear association test is
+
+.. math::
+
+\sum_k r_i c_j T_{ij}
+
+where :math:`r_i` and :math:`c_j` are row and column scores.  Often
+these scores are set to the sequences 0, 1, ....  This gives the
+'Cochran-Armitage trend test'.
```

Comment:
scores sounds a bit like weights in cohen's kappa http://statsmodels.sourceforge.net/devel/generated/statsmodels.stats.inter_rater.cohens_kappa.html#statsmodels.stats.inter_rater.cohens_kappa
but I never looked enough at Cochran-Armitage trend test or similar